src/PluginClient.php

```diff
@@ -62,9 +65,20 @@ public function __construct($client, array $plugins = [], array $options = [])
      */
     public function sendRequest(RequestInterface $request)
     {
-        $promise = $this->sendAsyncRequest($request);
+        if (!($this->client instanceof HttpClient)) {
```

Comment:
It doesn't matter IMO. If we emulate a sync client as well, this condition and call becomes unnecessary.